.github/workflows/worker.yml

```diff
@@ -45,4 +66,6 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: "${{ inputs.artifact_prefix }}-${{ env.HASH }}"
```

Comment:
can you add an `if` here - so that this only happens if the `artifact_prefix` is set?

Reply:
Done